packages/theme-default/src/components/LinkCard/index.tsx

```diff
@@ -0,0 +1,43 @@
+import styles from './index.module.scss';
+import ArrowRight from '@theme-assets/arrow-right';
+
+interface LinkCardProps {
```

Comment:
no `icon` props compare to `<Card />`?

Reply:
Because of #1506, the parameter design mainly refers to the `Card` and `LinkCard` components of astro. And `LinkCard` in astro has not `icon`.

Reply:
Fine, we can add `icon` if it's needed in the future.